hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -313,7 +314,12 @@
           for (int j = 0; j < numMetaReplicas; j++) {
             RegionInfo meta = RegionReplicaUtil.getRegionInfoForReplica(
                                     RegionInfoBuilder.FIRST_META_REGIONINFO, j);
-            ServerName metaLocation = MetaTableLocator.waitMetaRegionLocation(master.getZooKeeper(), j, 1);
+            ServerName metaLocation = null;
+            try {
+              metaLocation = MetaTableLocator.waitMetaRegionLocation(master.getZooKeeper(), j, 1);
+            } catch (NotAllMetaRegionsOnlineException e) {
+              //Should ignore this Exception for we don't need to display rit meta region info in UI
```

Comment:
Should ignore this Exception for we don't need to display rit meta region info in UI
=>
Should ignore this Exception as we don't need to display rit meta region info in UI

Use 'as' instead of 'for'.

And mind explaining a bit more on why we do not need to dispay rit meta region info? And what is going on if metaLocation is null?

Reply:
Ok! Thansk for your comments